addons/l10n_ar_website_sale/views/templates.xml

```diff
@@ -2,6 +2,7 @@
 <odoo>
 
     <template id="partner_info" name="Argentinean partner">
+        <t t-set="partner_sudo" t-value="partner_sudo or partner"/>
```

Comment:
It means this template is sometimes called with partner_sudo and sometimes with partner.
I would fix the calls to make sure the values are always coherent (partner_sudo). This change is hiding things under the carpet and making it more diffcult to debug in the future.

Reply:
Hello @arj-odoo :wave: ,
I have done the change but as we are fixing this in stable so do you think it might break some customization in customer's db ?